app/src/main/java/com/gruutnetworks/gruutsigner/util/AuthGeneralUtil.java

```diff
@@ -20,7 +22,18 @@ public static String getTimestamp() {
     }
 
     /**
-     * 32 byte Nonce를 Base64로 인코딩하여 반환
+     * 메세지의 timestamp를 기준으로 하여 유효한 메세지인지 확인한다.
+     * @param msgTimestamp  메세지에 포함된 timestamp(UNIX timestamp)
+     * @return 메세지의 validity
+     */
+    public static boolean isMsgValid(String msgTimestamp) {
```

Comment:
```suggestion
    public static boolean isMsgInTime(String msgTimestamp) {
```
함수 이름만 바꾸면 더 좋겠어요. 메시지가 유효하지 않는 경우는 시간 뿐만이 아닐테니까요.
혹은 isMsgValid라는 함수를 만들고, 그 안에서 
- 시간 검사
- 헤더 검사
- 그 외에 미래에 수행할 검사를 추가하면 함수 이름을 저렇게 지어도 좋겠죠.
제안하는대로 구현하는 경우에도 각각의 검사를 함수로 쪼개면 inMsgInTime 이라는 함수는 있는게 낫겠지만요..